Description: Remove "from xxx" from the files so we can better de-duplicate them.
Author: Christoph Berg <myon@debian.org>
Forwarded: not-needed

--- a/sql/scripts/build-extension-update-files.pl
+++ b/sql/scripts/build-extension-update-files.pl
@@ -266,7 +266,7 @@ sub write_script {
     || die "ERROR: failed to create '$output_directory/pgrouting-pgrouting--$old_version--$new_version.sql' : $!\n";
 
     my $header = "-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
--- pgRouting extension upgrade from $old_version to $new_version
+-- pgRouting extension upgrade to $new_version
 -- generated by tools/build-extension-update-file
 -- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 ";
